src/electron/index.js

```diff
@@ -130,5 +135,19 @@ app.on("open-url", (event, url) => {
 	broadcastURL(deeplinkingUrl);
 });
 
+app.on("before-quit", (e) => {
+	if (isDev) {
+		find("port", 3000).then(
```

Comment:
These electron events do not seem to wait for promises. The `concurrently` package has a parameter that seems to handle this` --kill-others`, so add it into the `dev` script in the ` package.json` file.